### DISCUSSION TITLE
Non Responsive Gutenberg "NavMenu" Block

### DISCUSSION THREAD

**Discussion by Blockheld** [WordPress/gutenberg#47778] [Q&A] [answered]
Hey there, I have a question about the "NavMenu" Block.
I have the problem, that the block doesn't adapt to different screen sizes.
I get there weird looking Breakpoints inside my titles. 
Could anyone tell me if I am just not having the right box checked, or is it something I need to fix with other magic?

<img width="726" alt="image" src="https://user-images.githubusercontent.com/124581389/216968625-c76ce3db-1647-4ca5-a769-2722efa1bf99.png">

**Comment by ghost** [comment, marked as answer]
<img width="287" alt="Screenshot 2023-03-06 at 4 44 09 PM" src="https://user-images.githubusercontent.com/118618474/223159197-5cf24ae6-dbcd-4c64-b01e-b8f93b13cb13.png">

In block settings you can specify if you want it to wrap on multiple lines or if you want to collapse the menu. I think in this case, with quite a lot of items, you could go for either always collapsing it, or wrapping it. My advice however would be to reorganize your menu. For example by combining some items in a dropdown or by reducing the space between menu items.
